tests/foreman/cli/test_activationkey.py

```diff
@@ -1578,8 +1578,7 @@ def test_positive_view_subscriptions_by_non_admin_user(module_manifest_org):
 
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
-@pytest.mark.skip_if_not_set('repos_hosting_url')
```

Comment:
removed this line because it wasn't actually being evaluated correctly. we can do this better later with selective validation